src/dir.props

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
+
+  <PropertyGroup>
+    <CoreHostOutputDir>$(BaseOutputRootPath)corehost\</CoreHostOutputDir>
+    <CoreHostLockedDir>$(CoreHostOutputDir)locked\</CoreHostLockedDir>
+    <SharedFrameworkSourceRoot>$(IntermediateOutputRootPath)sharedFramework\framework</SharedFrameworkSourceRoot>
```

Comment:
These seem to be global property in the BuildContext object of the old build system, but taking a closer look 'CoreHostOutputDir' is used both in src and test, the other ones can be moved under src.